main.tf

```diff
@@ -61,6 +61,12 @@ resource "aws_db_instance" "main" {
   db_subnet_group_name = "${var.name_prefix}-subnet-group"
 
   tags = merge(var.tags, { "Name" = "${var.name_prefix}-db" })
+
+  lifecycle {
+    ignore_changes = [
+      deletion_protection
+    ]
+  }
```

Comment:
Dette gjør at man ikke kan slå av eller på `deletion_protection` etter resursen er lagd/man bruker denne versjonen av modulen.

Å endre defaulten til `true` burde være mer enn nok med tanke på utfallet du ønsker i beskrivelsen til PRen.

Er det noen spesifikk grunn til at du ønsker å ignorere endringer?

Reply:
Det var for å forhindre noen å sette den tilbake til false (vi har noen PR-er på å sette deletion_protection eksplisitt til true). Var også usikker på hva som skjer om noen bare sletter terraform-koden som kaller denne modulen,  men når jeg tenker meg om burde ikke det ha noen betydning. Terraform destroy vil bare feile. 🤔

Og vi vil jo at man skal kunne eksplisitt sette `deletion_protection` til false, så tenker jeg bare fjerner denne.